### PR TITLE
IDVA6-1239 dashboard updates when ACSP is suspended

### PR DESCRIPTION
**Pull request by bmchugh-ch**
**Link to Jira**
https://companieshouse.atlassian.net/browse/IDVA6-1239

**Change Description**
When an ACSP has a status of suspended
- Warning text appears on dashboard
- links to verify someone's identity and file as an authorised agent are removed

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/views/partials/dashboard/file_as_an_authorised_agent.njk [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
+                </strong>
+            </h2>
+        {% else %}
+            <h2 class="govuk-summary-card__title govuk-!-margin-top-0">
+                {{lang.file_as_an_authorised_agent}}
+                <strong class="govuk-tag govuk-tag--blue govuk-!-margin-top-2">
+                    {{lang.coming_soon}}
+                </strong>
+            </h2>
+        {% endif %}
```
Comment (tferns-ch) [inline src/views/partials/dashboard/file_as_an_authorised_agent.njk:18]: Could maybe nest just the strong element? So we don't duplicate the `h2` and `file_as_an_authorised_agent` text?

```suggestion
        <h2 class="govuk-summary-card__title govuk-!-margin-top-0">
            {{lang.file_as_an_authorised_agent}}
            {% if agentStatus === AcspStatus.SUSPENDED %}
                <strong class="govuk-tag govuk-tag--red govuk-!-margin-top-2">
                    {{lang.cant_file_suspension}}
                </strong>
            {% else %}
                <strong class="govuk-tag govuk-tag--blue govuk-!-margin-top-2">
                    {{lang.coming_soon}}
                </strong>
            {% endif %}
        </h2>
```

**Review by tferns-ch — APPROVED** [review]
nice one, just a small suggestion

File: src/views/partials/dashboard/file_as_an_authorised_agent.njk (hunk 1 above)
Comment (bmchugh-ch) [reply src/views/partials/dashboard/file_as_an_authorised_agent.njk]: Thanks, updated. The h2 is no longer duplicated. Might be better to keep the {{lang.file_as_an_authorised_agent}} duplicated as is should become a link in the future but not when the ACSP is suspended.

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**